_posts/2017-01-27-welcome-to-vital-literature-review.markdown

```diff
@@ -1,6 +1,43 @@
 ---
-layout: post
-title:  "Welcome to the Literature Review site of VITAL."
-date:   2017-01-27 12:18:56 -0500
+layout:    post
+title:     Welcome to the Literature Review site of VITAL
+date:      2017-01-27 12:18:56 -0500
+permalink: /welcome-how-to/
 ---
-On this site, you'll find all the essential papers any deep learning expert should read.
```

Comment:
replace deep learning expert by 
papers the VITALab members should be aware of.